docs/package.json

```diff
@@ -29,17 +29,17 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.30",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
-    "@material-ui/core": "5.0.0-alpha.32",
+    "@material-ui/core": "^5.0.0-alpha.32",
```

Comment:
Now that I think about it: Pinning to exact match is the correct thing to do. It's unclear though why these weren't updated on the last release to `5.0.0-alpha.33`